### ISSUE TITLE
_vimfiles on windows

### ISSUE THREAD

**Issue by FernandoBasso** [SirVer/ultisnips#426]
I see that UltiSnips is not expanding my custom snippets, which UltiSnips automatically puts in
`C:\Users\me\_vimfiles\UltiSnips`. Shouldn't it be `vimfiles` without the leading underscore?

It expands any of the snippets of the package `vim-snippets`, just the custom ones that don't expand and it seems to me that it is because of that underscore in front of `vimfiles`.

I have no problems at all on Linux.

**Comment by SirVer** [comment]
I do not have access to a windows computer to test, but I know for sure that two years ago the testsuite passed on windows and needed the _ . Maybe that has changed? 

You could try getting the testsuite running on windows again if you are so inclined. Or you try searching for both _vimfiles and vimfiles.

**Comment by Oberon00** [comment]
That should be fixed since a3e654fc5486beae7b2c787315cc2aba850d95d2 (6 Apr 2014). Are you using the latest version of UltiSnips?

**Comment by FernandoBasso** [comment]
Two days ago, I just entered `vimfiles/bundle` and ran `git submodule add git://github.com/SirVer/ultisnips.git`, so I believed it was the latest version.

But I see that `snippet_manager.py` really has `_vimfiles` rather than `vimfiles`. I guess I must have cloned a wrong repository, because I deleted it and cloned it again (from the correct repository), and it works out just as expected now.

Sorry for having taken you time because of my foolish mistake. I'm loving UltiSnips.
